packages/manager/modules/hub/src/components/billing-summary/index.scss

```diff
@@ -0,0 +1,69 @@
+.manager-hub-billing-summary {
```

Comment:
Could some styles be put in common with other features ?

Reply:
yes, i was thinking MANAGER-4306 was here to handle the style refactoring and merging of all the hub components